pipes/negativ_pipe.py

```diff
@@ -0,0 +1,14 @@
+from pipes.pipe import Pipe
+import numpy as np
+
+class NegativPipe(Pipe):
+    def pipe(self, img):
+        whiteframe = np.ones(img.shape, sdtype=np.uint8) * 255
```

Comment:
I get an error executing the filter here: I think you mean `dtype=np.uint8` instead of `sdtype=np.uint8`